proto/proto/store/rpc.proto

```diff
@@ -164,19 +164,22 @@ message AccountProofResponse {
         // Account storage data
         AccountStorageDetails storage_details = 2;
 
-        // Account code; empty if code commitments matched or none was requested
+        // Account code; empty if code commitments matched or none was requested.
         optional bytes code = 3;
 
         // Account asset vault data; empty if vault commitments matched or the requester
         // omitted it in the request.
         optional AccountVaultDetails vault_details = 4;
     }
 
-    // Account ID, current state commitment, and SMT path
-    account.AccountWitness witness = 1;
+    // The block number at which the witness was observed.
+    blockchain.BlockNumber block_num = 1;
```

Comment:
nit: I'd mention that this is not only about the witness but also about account details.